### DISCUSSION TITLE
Still not get approval for student developer's pack. I applied too much times for Student developer's pack. Can any one will help me?

### DISCUSSION THREAD

**Discussion by Rocky-2003** [community/community#60391] [GitHub Education]
### Hi everyone,

Hi everyone, can anyone will help to get approval. I applied to much times  for student developer Pack  but still get not approved. what should I do?
MODERATOR EDIT: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
![Uploading 3333.jpg…]()
![Uploading WhatsApp .jpg…]()

**Comment by samus-aran** [comment]
Hi @Rocky-2003 

Common pre-processing warnings are because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
